pyproject.toml

```diff
@@ -23,7 +23,6 @@ classifiers = [
 ]
 
 dependencies = [
-    "dbt-core>=1.5",
```

Comment:
As far as we still have functionality within a package that uses `dbt-core` (conditionally), did you consider making it a dev dependency?

Reply:
Good idea, added it to the dev dependencies 👍